contrib_bots/lib/github.md

```diff
@@ -0,0 +1,68 @@
+# Zulip Github API
```

Comment:
I think you should make it clear who your audience is for this document.  My understand is that it's for future bot developers to use if they want to use your library, correct?  I think an explanatory paragraph here would be helpful.